addons/website/models/ir_http.py

```diff
@@ -379,6 +380,26 @@ def get_frontend_session_info(self):
             })
         return session_info
 
+    @classmethod
+    def _is_allowed_cookie(cls, cookie_type, cookie_name='undefined'):
+        result = super()._is_allowed_cookie(cookie_type, cookie_name)
```

Comment:
The logic of this method is a bit odd, you call the super but you never check the value and assume `result` is `True`.
Either you take the lead and don't call super at all, or you change a bit to call `super` if you are not in a website-supported case ("required" and "optional"). Let's assume I create a module that adds the type `performance`, this method would force it to `required`.